samples/fb-passthrough-sample/export_presets.cfg

```diff
@@ -0,0 +1,227 @@
+[preset.0]
+
+name="Quest"
+platform="Android"
+runnable=true
+advanced_options=true
+dedicated_server=false
+custom_features=""
+export_filter="all_resources"
+include_filter=""
+exclude_filter=""
+export_path="build/quest/XRPassthroughSample.apk"
```

Comment:
The `export_path` can be made relative to the `samples` directory so all samples output to the same location for easy access.